data/data.go

```diff
@@ -65,3 +65,71 @@ type StringModifiers struct {
 	I        bool `json:"i"` // for regex
 	S        bool `json:"s"` // for regex
 }
+
+type Keyword struct {
+	Name string
+}
+
+type Expression struct {
+	Left        interface{}
+	Operator    string
```

Comment:
What if we had a type for these operators, such as:

```go
type Operator string

const (
    OperatorMatches Operator = "matches"
    OperatorAt Operator = "at"
    ...
)
```
If that existed in `data.go`, then people could use it as a sort of enum rather than relying on raw string matches.

Reply:
Good idea, I'll do that.

Reply:
While I'm at it, I'll also do that for keywords.